### ISSUE TITLE
CLI procedure is sometimes easy to miss

### ISSUE THREAD

**Issue by tahliar** [theforeman/foreman-documentation#457]
Currently the docs use Procedure for the web UI steps, followed by For CLI Users. However, if the web UI procedure is quite long, the CLI procedure can get buried and people might not know it's there. 

Could we do something like this: 

```
Doing the Task

Use this procedure to do the task. To use the CLI instead of the web UI, see the xref:cli-doing-the-task[]. 

.Procedure

<steps>

[[cli-doing-the-task]]
.CLI Procedure

<steps>
```

I'm happy to do this myself, but wanted to get your thoughts first.

**Comment by melcorr** [comment]
+1 from me, I think that we standardized it to .For CLI users because someone did that once and we all followed.

**Comment by tahliar** [comment]
Fixed in PR #461
